src/main/java/org/fenixedu/academic/domain/DegreeHelper.java

```diff
@@ -19,16 +19,31 @@
 package org.fenixedu.academic.domain;
 
 import java.util.Optional;
+import java.util.stream.Collectors;
 
+import org.fenixedu.bennu.core.domain.groups.PersistentGroup;
 import org.fenixedu.bennu.core.groups.Group;
 import org.fenixedu.bennu.core.groups.NobodyGroup;
+import org.fenixedu.bennu.core.groups.UserGroup;
 
 public class DegreeHelper {
     public static void setCanManageThesis(Degree degree, Group group) {
         degree.setThesisManager(group.toPersistentGroup());
     }
 
     public static Group getCanManageThesis(Degree degree) {
```

Comment:
This method should keep its semantic, this is simply an access method for the configurable thesis manager group of a degree, adding coordinators here defeats the purpose of a configurable group